src/components/tutorial-card/helpers/format-tutorial-card.ts

```diff
@@ -8,16 +8,17 @@ import { TutorialCardPropsWithId } from '../types'
 
 export function formatTutorialCard(
   tutorial: ClientTutorialLite,
-  collectionSlug: string
+  collectionSlug?: string
 ): TutorialCardPropsWithId {
+  const safeCollectionSlug = collectionSlug || tutorial.defaultContext.slug
```

Comment:
Might be worth a note here that there can be collection slugs which aren't the default collection 👌 I could see getting confused if I didn't know that!

Reply:
But perhaps with these cards we just always use the default as the link?

Reply:
✅ Great call 💖 just added!